### ISSUE TITLE
A way to configure the loading screen while in-game

### ISSUE THREAD

**Issue by Gabyfle** [Gabyfle/gLoad#18]
Add a way to configure the loading screen while in-game.

- [ ] Only administrators
- [ ] Preview changes
- [ ] Full Lua code (no HTML interpreter) : everything MUST be done in Lua to avoid browser incompatibilities

**Comment by ReturnEnd** [comment]
Admin panel can be opened from Steam Web browser, is it really worth it ?

**Comment by Gabyfle** [comment]
You're right but actually, I don't know if the Steam browser is up to date with the latest technologies (which engine this browser use ?). I'll investigate.
